sync.py

```diff
@@ -157,6 +162,7 @@
     parser.add_argument('--password')
     parser.add_argument('--limit', type=int, default=25)
     parser.add_argument('--verbose', action='store_true')
+    parser.add_argument('--allowinvalidssl', action='store_true')
```

Comment:
I would add a `default=True` param that way users have to explicitly disable it and it allows for some shorter code above - mainly we can just pass the param and not worry about an if-else block. It also allows us to be more explicit which is a good thing

Reply:
So this would become `parser.add_argument('--allowinvalidssl', action='store_true', default=False)`